src/pages/workspaces/List.js

```diff
@@ -170,6 +171,12 @@ export class WorkspaceList extends Component {
       ])
     ])
   }
+
+  componentDidUpdate() {
+    const { filter, listView, itemsPerPage, pageNumber } = this.state
+
+    StateHistory.update({ filter, listView, itemsPerPage, pageNumber })
```

Comment:
In another implementation, I also stored the data (`workspaces` in this case). That meant that when a user hit the back button, they would instantly see the "stale" results while they waited for the new results to load (which is often unchanged). It makes for a very responsive experience.

Reply:
I'm only doing this for entity types, at least for now